packages/components/src/OverlayTrigger/OverlayTrigger.component.js

```diff
@@ -93,6 +93,11 @@ export default class OverlayTrigger extends React.Component {
 			rootClose: true,
 			trigger: 'click',
 		};
+		const tooltipProps = {
+			onMouseOver: this.props.onMouseOver,
+			onMouseOut: this.props.onMouseOut,
+			onFocus: this.props.onFocus,
```

Comment:
The span is not the element that is focused. This is never called, we should not pass this one to the span.
BTW there is an issue with tooltip display with keyboard navigation. On focus it appears, but never disappear

Reply:
fixed, events are applied to button